docs/Execution.md

```diff
@@ -116,14 +116,25 @@ If you have not installed the drivers, here are some links that may help:
 * For [AWS P2 GPUs](https://aws.amazon.com/blogs/aws/new-p2-instance-type-for-amazon-ec2-up-to-16-gpus/2)
 * For [Google Cloud GPUs](https://cloud.google.com/compute/docs/gpus/add-gpus)
 
-**Step 2**: For Debian/Ubuntu users, install `nvidia-docker` by following the instructions 
-[here](https://github.com/NVIDIA/nvidia-docker).
+**Step 2**: For Debian/Ubuntu users, install `nvidia-docker` for your version of Docker. For instance, on Ubuntu, the following commands install Docker 18.03.1 and the appropriate version of `nvidia-docker`:
 
-**Step 3**: Test your setup. Run:
+    sudo apt-get install docker-ce=18.03.1~ce-0~ubuntu
+    sudo apt-get install nvidia-docker2=2.0.3+docker18.03.1-1 nvidia-container-runtime=2.0.0+docker18.03.1-1
+    sudo systemctl daemon-reload
+    sudo systemctl restart docker
 
-    sudo nvidia-docker run --rm nvidia/cuda nvidia-smi
+**Step 3**: Test your setup by checking if Docker can find GPUs. Run:
 
-**Step 4**: Run this command, which tests that nvidia-smi is working inside of Docker:
+    sudo docker run --runtime=nvidia --rm nvidia/cuda nvidia-smi
+    
+You should see something similar to before:
+
+    Wed May 24 19:03:55 2017``
+    +-----------------------------------------------------------------------------+
+    | NVIDIA-SMI 367.48                 Driver Version: 367.48                    |
+    ...
+
+**Step 4**: Run this command, which tests that `nvidia-smi` is working inside of Docker:
```

Comment:
Replace: "Test whether `nvidia-smi` is working inside Docker through CodaLab:"

Reply:
(Isn't this just the same as the previous step, but just through CodaLab rather than through docker directly?)

Reply:
Yeah, I typo'd